RecoTauTag/Configuration/python/DumpTauVariables_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+dumpTauVariables = cms.EDProducer(
```

Comment:
file name should normally match the module instance name (module instance name starts with lower case, according to CMSSW style).
So, it would be better to rename this to dumpTauVariables_cfi.py